### DISCUSSION TITLE
power supply?

### DISCUSSION THREAD

**Discussion by jkmhb** [tagyoureit/nodejs-poolController#579] [App and O/S Config Questions]
I have a smart fan and 8 relay hat, I have a home automation hat on order, all sequent. I read in the wiki that a 24 vdc power supply was used, but in the doc for the hat, on the sequent website  it says 24vac. Which power supply should I order?

**Comment by rstrouse** [comment]
The MEGA-BAS operates with either 24vdc or 24vac.  You can power with either one.  In fact you can use a AC to DC rectifier to power relays if you only have a 24vac power supply.

**Comment by jkmhb** [comment]
Thank you! I guess their doc is incorrect

**Reply by rock-crusher** [reply]
Not sure why it’s not mentioned in their power requirement documentation, but it is clearly marked on the card itself.

![Bas 24V AC-DC](https://user-images.githubusercontent.com/94593954/187051932-586e06c4-5fb1-492c-8c9f-279e362134c8.JPG)

**Comment by rstrouse** [comment]
Hey I just re-read your initial post.  Make sure you are not getting the Home Automation Card.  You want the Building Automation Card.  The relays are only normally open on that card and the inputs require external circuitry for temp sensors.  The Building Automation Card is a much better fit.

**Reply by jkmhb** [reply]
Thank you both! I double checked- I did order the right card. It is all very confusing.

**Reply by jkmhb** [reply]
.

**Comment by jkmhb** [comment]
I received my card yesterday. it reports hardware verion 4.0, it says 4.1 on the card, It also says on the card 24VAC and there is no battery installed.
I did order the Building Automation card.
![IMG_0061](https://user-images.githubusercontent.com/50624053/187436474-28a3e2ca-3514-41b4-83a1-febc2e0e2f10.jpg)

**Reply by tagyoureit** [reply]
That is definitely a new version of the card.  It never has come with a battery, but it isn't clear from the website.  And there is now a button and only 1 rs485 connector.  Where the 2nd rs485 connector used to be there is now a 5v/gnd/in (or iw?), and the big green capacitor(?).  

And like you mention the input label only says 24vac.  I would email Sequent and get clarification on the input before hooking 24vdc up to it.

**Reply by rock-crusher** [reply]
Good point tagyoureit, this is definitely different from the previous version. The screen printing next to the input connector was listed at 24VAC/DC (previous version). Now, this newer one jkmhb has only states 24VAC? What the heck? I agree, very confusing and why all the changes, be interesting to hear what jkmhb learns if he contacts them.

**Reply by rstrouse** [reply]
Ok after looking at this card long and hard to determine what changed with it.  It appears that they replace the second RS485 connector with a 1-wire interface.  The capacitor appears to be an addition to smooth and stabilize 24v power over the bridge rectifier.  Still interested in what Sequent has to say about the revisions.  If the above is true then that isn't such a bad addition to the card.

**Reply by tagyoureit** [reply]
Oh! 1W.  I was close but good eyes.

**Reply by jkmhb** [reply]
I have contacted Sequent. We will see what they say...

**Reply by jkmhb** [reply]
They said 24vdc will work. They did not answer other questions about card, I asked twice.

**Reply by rstrouse** [reply]
I figured it would still digest 24vdc since they needed a bridge rectifier in there somewhere.   There have been no software changes on their site for this so I am guessing it is all still compatible.  I did see that they did have a firmware update but I don't know what that entails.

**Reply by Telemachus1977** [reply]
I guess they assumed that since AC requires more complex conversion, they would just print that one on the board.

Saved them half a Yuan not printing DC.
